movie_search/migrations/0009_alter_movie_vote_count_alter_moviegenre_genre_id_and_more.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 4.2.9 on 2024-04-07 06:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I don't see model changes for those, were those missing migration files from previous changes?